rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -1153,7 +1153,12 @@ void ppu_thread::dump_regs(std::string& ret) const
 			}
 		}
 
-		fmt::append(ret, "\n");
+		if (usz index = ret.find_last_not_of(' '); index != umax)
+		{
+			ret.resize(index + 1);
+		}
```

Comment:
Can't you just use fmt::trim ?

Reply:
or maybe a new method called "trim_back" or whatever is commonly used.

Reply:
Okay.